plugin/figure/src/main/java/io/deephaven/figure/FigureWidgetTranslator.java

```diff
@@ -108,9 +123,8 @@ private FigureDescriptor translateFigure(FigureWidget f, SessionState sessionSta
             }
             i++;
 
-            SessionState.ExportObject<Table> tableExportObject = sessionState.newServerSideExport(table);
-            clientFigure.addTables(TableServiceGrpcImpl.buildTableCreationResponse(
-                    TableReference.newBuilder().setTicket(tableExportObject.getExportId()).build(), table));
+            final Reference export = exporter.newServerSideReference(table);
+            clientFigure.addTables(ExportUtil.buildTableCreationResponse(export.id(), table));
```

Comment:
I'm not against removing `TableInfo` (or having explicit RPCs to retrieve the info from a ticket). In which case, I think your suggestion from earlier will be to prefer `  string field_type = 3;`...

Reply:
@mofojed does the web UI use

```
/*
 * Lightly describe a Table.
 */
message TableInfo {
  // Schema as described in Arrow Message.fbs::Message.
  bytes schema_header = 1;

  // Whether or not this table might change.
  bool is_static = 2;

  // The current number of rows for this table.
  sint64 size = 3 [jstype=JS_STRING];
}
```?

(part of ApplicationService.listFields RPC)

Reply:
Ah, ETCR != TableInfo (I was mixed up for a bit). But the questions are still valid.

Reply:
Looks like not setting ETCR tables in Figure breaks the current web UI - just an FYI, but I'm assuming we are OK with that since Bender has other type-breaking changes he's working to fix too.

Reply:
I have that partly fixed locally now (as above, we discussed this briefly) - client-created plots work using this new setup, but server-created ones aren't quite fetching properly yet, should have it resolved today.